File: Python Assignments/untitled folder/primeNumberChecker.java
import java.util.Scanner;

public class primeNumberChecker {
    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int userNumber = userInput.nextInt();

        boolean isPrime = true;
        for (int number = 2; number <= userNumber / 2; number++) {
            if (userNumber % number == 0) {
                isPrime = false;
                break;
            }
        }
        if (isPrime) {
            System.out.printf("The number %d is a prime number \n", userNumber);
        }
        else{
            System.out.printf("The number %d is not a prime number \n", userNumber);
        }

        System.out.printf("All prime numbers between 1 and %d are: ", userNumber);
        for (int numbers = 1; numbers < userNumber; numbers++){
            boolean prime = true;
            for (int number = 2; number < numbers/2 + 1; number++ ){
                if (numbers % number == 0){
                    prime = false;
                    break;
                }
            }
            if (prime){
                System.out.print(numbers + " ");
            }
        }
        System.out.println();
    }
}

File: Python Assignments/untitled folder/prime_number_checker.py
user_number = int(input("Enter a number: "))

for number in range(2, user_number//2 + 1):
    if number % user_number == 0:
        primt = False
prime = True
if prime == True:
    print(f"The number {user_number} is a prime number")
else:
    print(f"The number {user_number} is not a prime number")


print(f"All prime numbers between 1 and {user_number} are: ", end ="")
for numbers in range(2, user_number):
    prime = True
    for number in range(2, numbers//2 + 1):
        if numbers % number == 0:
            prime = False
            break

    if prime:
        print(numbers, end = " ")

print()